packages/web/src/scss/components/Dropdown/index.html

```diff
@@ -1,30 +1,219 @@
 {{#> layout/plain }}
 
-<div class="Dropdown is-open">
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#info" />
-    </svg>
-    <span>Information</span>
-  </a>
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#link" />
-    </svg>
-    <span>More links</span>
-  </a>
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#profile" />
-    </svg>
-    <span>Profile</span>
-  </a>
-  <a href="#" class="d-flex">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#help" />
-    </svg>
-    <span>Help</span>
-  </a>
-</div>
+<article>
+  <span> void space </span>
+  <button
+    data-toggle="dropdown"
+    data-target="DropdownDemo1"
+    data-reference="toggle"
```

Comment:
The README says `toggle` is the default value. Can we leave it off here then?

Reply:
Whatever, i just put it here for example.